cfpb/consumerfinance.gov
# Remove super button

See https://[GHE]/CFGOV/platform/issues/4492

https://www.consumerfinance.gov/compliance/compliance-resources/other-applicable-requirements/privacy-notices/
https://www.consumerfinance.gov/complaint/

Were the only pages that had `a-btn__super` and these have been moved to regular buttons.

## Removals

- Remove super buttons.


## How to test this PR

1. `python cfgov/manage.py migrate`
2. Search for a call to action block in wagtail. Check any of those pages and see that their isn't a button size option in the call to action molecule any longer.
